src/components/main-layout/Layout.tsx

```diff
@@ -62,14 +64,16 @@ const MainLayout: React.FC<MainLayoutProps> = ({
                 </VisibilityControl>
 
                 <Content className={styles.centerPanel}>{centerPanel}</Content>
-                <VisibilityControl excludedPages={[9, 10]}>
+                <VisibilityControl
+                    conditionalRender={layout !== LayoutType.WideScreen}
```

Comment:
`LayoutType.WideScreen` and `LayoutType.FullScreen` to me imply more about the aspect ratio of the application rather than a layout... could they maybe be renamed? `FullDetail` or `FullSimulationPanel` or something?